### DISCUSSION TITLE
Are the follow modes and target modes tested with Godot 4.3 ?

### DISCUSSION THREAD

**Discussion by deezgamezdev** [ramokz/phantom-camera#371] [Question / Help Wanted] [answered]
I was using your plugin with no problems at all. I had a little one but you guys even helped me with that and it was fixed.

Recently I updated to Godot 4.3 but now whenever I load the scene the camera has a weird rotation on load, but on the editor preview everything is fine ?

There is a small footage demonstrating...

Thanks once again guys, and definitely keep up with this amazing plugin.

P.S: The orbiting code is taken from the example on the docs.

https://github.com/user-attachments/assets/7d29d323-47f0-4f53-af76-23ab73725b9f

**Comment by ramokz** [comment]
That looks strange.

I've been running tests in Godot 4.3 since its first pre-release build, currently on the stable build, and haven't noticed any new issues.

Not sure what's going on in your scene, but happy to have a look if you want to share a minimal-reproducible project.

**Reply by deezgamezdev** [reply]
I actually found out today that if I make the PhantomCam top level I dont have this issue, but its weird because it shouldnt affect the look at function. I tried with other camera systems and the targets worked..

But nevermind, thanks alot anyways, I figured out and this way I can keep using your plugin happily .

**Comment by deezgamezdev** [comment]
hmmm... sorry but the follow works correctly but now the orbiting part doesnt work.
I am using exactly the same code in the docs and with the camera top level I get this output.

https://github.com/user-attachments/assets/013d3ea3-6928-4524-b036-d1e9c845c29e

**Reply by deezgamezdev** [reply, marked as answer]
Also figured out a way around it, if I remove the offset from the follow mode this doesnt happen. I just need to adjust my zoom values. Thanks again and sorry for all the trouble.
